Quantum-Control-Applications/Superconducting/Pound Drever Hall Mesurement with Octave/Cross_correlation_noise_analyzer.py

```diff
@@ -0,0 +1,239 @@
+"""
+Cross_correlation_noise_analyzer.py:
+
+Version: 0.1
+
+"""
```

Comment:
I would put here the basic information from the readme file (and remove filename and version)

Reply:
yes! I just did not get to that yet. thanks

Reply:
FYI, if a PR is not to be merged (not ready yet), then it's better to open it as a "draft"